src/components/ContentSidebar/__tests__/__snapshots__/SidebarVersions-test.js.snap

```diff
@@ -53,60 +53,28 @@ ShallowWrapper {
   Symbol(enzyme.__node__): Object {
     "instance": null,
     "key": undefined,
-    "nodeType": "class",
+    "nodeType": "function",
     "props": Object {
-      "children": <VersionHistoryLink
-        className=""
-        onClick={undefined}
-        versionCount={2}
-      />,
       "className": "",
-      "isOpen": true,
+      "onClick": undefined,
+      "versionCount": 2,
     },
     "ref": null,
-    "rendered": Object {
-      "instance": null,
-      "key": undefined,
-      "nodeType": "function",
-      "props": Object {
-        "className": "",
-        "onClick": undefined,
-        "versionCount": 2,
-      },
-      "ref": null,
-      "rendered": null,
-      "type": [Function],
-    },
+    "rendered": null,
     "type": [Function],
   },
   Symbol(enzyme.__nodes__): Array [
     Object {
       "instance": null,
       "key": undefined,
-      "nodeType": "class",
+      "nodeType": "function",
       "props": Object {
-        "children": <VersionHistoryLink
-          className=""
-          onClick={undefined}
-          versionCount={2}
-        />,
         "className": "",
-        "isOpen": true,
+        "onClick": undefined,
+        "versionCount": 2,
       },
       "ref": null,
-      "rendered": Object {
-        "instance": null,
-        "key": undefined,
-        "nodeType": "function",
-        "props": Object {
-          "className": "",
-          "onClick": undefined,
-          "versionCount": 2,
-        },
-        "ref": null,
-        "rendered": null,
-        "type": [Function],
-      },
+      "rendered": null,
```

Comment:
@DanDeMicco not sure why these are rendering null now, I just removed the <SidebarSection>